server/src/test/updateBiodata.service.test.ts

```
@@ -0,0 +1,190 @@
+        occupation: 'Doctor',
+        complexion: 'Medium',
+        religion: 'Islam',
+        bloodGroup: 'B+',
+      },
+    };
+
+    (PersonalAllDetailsModel.findOne as jest.Mock).mockResolvedValue(mockBiodata);
+
+    const updatedBiodata = {
+      ...mockBiodata,
+      personalInfo: updateData.personalInfo!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)